src/components/editor/editor-pane/status-bar/status-bar.tsx

```diff
@@ -27,11 +30,20 @@ export const createStatusInfo = (editor: Editor, maxDocumentLength: number): Sta
   charactersInDocument: editor.getValue().length,
   remainingCharacters: maxDocumentLength - editor.getValue().length,
   linesInDocument: editor.lineCount(),
+  wordsInDocument: wordsCount(editor.getValue()),
```

Comment:
So how accurate is this? How does it handle code blocks? How does it handle links? How does it handle raw HTML?

Reply:
The lib is not code specific. So it would count ANYTHING, that is separated by a space, which is useless imho.

The only usage that could be useful is, if we count the text-only content from the render output.

Reply:
Well actually it does not split only at spaces but recognizes CJK words etc. But it's right, that approach is counting every word in the markdown "code".

Reply:
Counting words in the HTML render output is a thing that we definitely should do in the long term, but I would rather locate such information into the "document statistics"-modal than in the editor-statusbar.

Reply:
That's basically the problem I mentioned in the related issue. There are problems ranging from people commenting large parts of the text and the counter would actually increase not decrease, and even when we used the rendered output it's quite likely that it would still count code blocks with it.

It's a wrong counter and I'm not sure if that is really helpful. If we really want to integrate it, we should make it clear, that this is a rough estimation (e.g. by adding a `~` in front of the number) as I can already see lots of bug reports flowing in, telling us that our counter is wrong (e.g. "Simple sentence with smiley at the end is counted wrong. There are 5 not 6 words. Example: `This is a simple sentence. :smile:`").